lib/settings.dart

```diff
@@ -360,4 +308,25 @@ class _SettingsScreenState extends State<SettingsScreen> {
       ),
     );
   }
-}
\ No newline at end of file
+}
+
+class SettingText extends StatelessWidget {
```

Comment:
I would suggest adding this class in a separate file. But would like to have @anushbhatia 's view on this as well.